### PR TITLE
fix: Form item has default height align with form size

### LINKED ISSUE #21468: form表单label错位
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://github.com/i7guokui/form-bug](https://github.com/i7guokui/form-bug)

### Steps to reproduce
直接点击'确定'按钮，显示校验不通过的提示信息，"类型"的label位置会跳动

### What is expected?
label位置不变

### What is actually happening?
label位置跳动

| Environment | Info |
|---|---|
| antd | 4.0.0-rc.5 |
| React | react |
| System | win10 |
| Browser | chrome 78 |

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

fix #21468

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Fix Form.Item with `help` align style.       |
| 🇨🇳 Chinese |    修复 Form.Item 设置 `help` 时的样式问题。        |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/form/demo/validate-other.md](https://github.com/ant-design/ant-design/blob/form-help/components/form/demo/validate-other.md)

### CODE REVIEW

**Comment by zombieJ** [comment]
Add one more div to align the center.

File: components/form/style/index.less [hunk 1]
```diff
@@ -123,7 +143,13 @@
 
   &-control-input {
     position: relative;
-    .clearfix;
+    display: flex;
```
Comment (afc163) [inline components/form/style/index.less]: 这句话把 https://github.com/ant-design/ant-design/issues/21586 搞坏了。

File: components/form/style/index.less [hunk 2]
```diff
@@ -67,7 +87,7 @@
       display: inline-flex;
       align-items: center;
       height: 100%;
-      max-height: @input-height-base;
+      height: @input-height-base;
```
Comment (yoyo837) [inline components/form/style/index.less]: 这里重复了一个`height`

Comment (afc163) [reply components/form/style/index.less]: 提个 issue。

**Pull request merged**